.prettierignore

```diff
@@ -0,0 +1 @@
+README.md
```

Comment:
Missing an endline. Created a PR to fix it.

Reply:
Go ahead 🙌